src/ucs/vfs/base/vfs_obj.c

```diff
@@ -31,10 +32,13 @@ typedef enum {
 typedef struct ucs_vfs_node ucs_vfs_node_t;
 struct ucs_vfs_node {
     ucs_vfs_node_type_t    type;
+    int16_t                refcount;
+    uint8_t                dirty;
```

Comment:
maybe add flags field?

Reply:
Did you mean to add enum with two UCS_BIT to try to save space?

Reply:
rename "dirty"->"flags" and define one bit "VFS_DIRTY" for now
since we'll probably have more flags

Reply:
I see. Thanks